### ISSUE TITLE
[ui][firefox] Right click on editor tab scrollbar shows native menu

### ISSUE THREAD

**Issue by geropl** [theia-ide/theia#4007]
When (accidentally) right-clicking the editor tab bars scrollbar the browser native context menu is shown. As a user I would expect that to be suppressed because it breaks the Theia experience:
![out](https://user-images.githubusercontent.com/32448529/50969144-bfd13e80-14dd-11e9-8cfb-60c1e83ae4cd.gif)

(Happens to me very often when trying to "Close all" editor tabs.

**Comment by AlexTugarev** [comment]
Chrome on Mac: same behavior

<img width="268" alt="Screen Shot 2019-03-22 at 10 01 43" src="https://user-images.githubusercontent.com/914497/54811710-92490400-4c89-11e9-89fd-3454426e8656.png">

**Comment by jankeromnes** [comment]
I agree, when you have many files open, that horizontal scroll bar is annoying (it takes so much space on Firefox Nightly no Mac).

Then again, I sometimes forget about some open files, because there is no other indicator that I have too many tabs open and they're overflowing.

VS Code also has just the horizontal scrollbar, but it's super thin on Mac so it's pretty easy to click on tab handles (and it's actually pretty hard to try and "accidentally" click the scrollbar).

TL;DR moving the scrollbar down and making it much thinner would help a lot.
